docs/docs/features/split-keyboards.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Split Keyboards
+sidebar_label: Split Keyboards
+---
+
+ZMK supports split keyboard setups where two or more keyboard parts (also called "sides" or "halves"), each with their own controller running ZMK, communicate with each other to work as a single keyboard device.
+
+:::note[Split communication protocols]
+Currently ZMK only supports split keyboards that communicate with each other wirelessly over BLE.
+As such, only controllers that support BLE can be used with ZMK split keyboards.
+
+Supporting split communication over wired protocols is planned, which would allow using ZMK split keyboards with non-wireless controllers.
+:::
+
+## Central and Peripheral Roles
+
+In ZMK splits, one keyboard part has the "central" role that receives key position and sensor events from the other parts that are called "peripherals."
+This part runs the necessary keymap logic to convert received events into HID events such as keycodes and then communicates with the connected host devices, e.g. over USB or bluetooth.
+
+As a result of this architecture the internal keyboard state (like active layers) is shared and is mainly handled by the central part.
```

Comment:
I think I was thinking about underglow/ext power etc. state since they just synced from the central. Even then, I think it would be fair to say "exclusively handled by the central" and would simplify understanding.

Reply:
I think I want to preserve the intention of "is shared" here, where I was trying to emphasize the "work as a single keyboard device" from above. Any suggestions on wording that better?

Reply:
On a second look I like the suggested version.